src/main/java/org/jitsi/stats/media/AbstractStatsPeriodicRunnable.java

```diff
@@ -62,39 +72,86 @@
     /**
      * The stats service to use for periodic reports.
      */
-    private StatsService statsService;
+    private final StatsService statsService;
 
     /**
      * Constructs <tt>AbstractStatsPeriodicRunnable</tt>.
      *
      * @param o the conference/call used to report statistics.
      * @param period the reporting interval.
      * @param statsService the stats service that will be serving this reporting
-     * @param conferenceName the conference name.
+     * @param conferenceJid the conference jid.
      * @param conferenceIDPrefix the conference prefix.
      * @param initiatorID the initiator.
      */
     public AbstractStatsPeriodicRunnable(
         T o,
         long period,
         StatsService statsService,
-        String conferenceName,
+        String conferenceJid,
```

Comment:
I thought the idea was to move away from jid stuff?

Reply:
jid is the only format that have all the information, check https://github.com/jitsi/jicofo/pull/585
And as to keep parsing of that information in one place and not spread it around it is here.

Reply:
If that is the case, I'd say let's either have separate `String` fields or use some `Jid` type here: what we have now is the requirements of what's in a Jid without the type safety.